mender/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 * **BREAKING CHANGES**:
   * This version of the Helm chart is not compatible with Mender versions older than `3.6.0`.
 
+## Version 5.0.2
+* Added `auditlogs.logRetentionSeconds` conf parameter for tuning auditlog settings
```

Comment:
Do we really need to release this as 5.0.2? Can't we simply release it as part of 5.1.0 and that's it?

Reply:
Yes, of course. I targeted it as 5.0.2 because of your comment :) 
> @oldgiova please target master-next so we can release it as part of 5.0.2.